qemu/tests/cfg/sev_hotplug_mem.cfg

```diff
@@ -0,0 +1,42 @@
+- sev_hotplug_mem:
+    type = sev_hotplug_mem
```

Comment:
hi Zixi, we may add some filters here, e.g.
only ovmf
only q35
only HostCpuVendor.amd

And the sev functionality enablement check.

But ...... it's OK for me too

Reply:
Thanks Zhenchao, I will add some filters on cfg, but I plan to only add sev functionality enablement checks to sev boot type cases. If the test points are not boot sev guests, I tend to not add host sev checks. Since it is a duplicate work, we will know host is not supported in the boot case.

Reply:
Update code.
Check with python3 ConfigTest.py --testcase=sev_hotplug_mem --guestname=RHEL.9.*   --platform=x86_64 --clone=no --profile=profile
 (1/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_ram.hotplug.q35: STARTED
 (1/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_ram.hotplug.q35: PASS (148.27 s)
 (2/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_ram.unused_memory.q35: STARTED
 (2/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_ram.unused_memory.q35: PASS (140.84 s)
 (3/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_file.hotplug.q35: STARTED
 (3/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.*.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_file.hotplug.q35: PASS (158.59 s)
 (4/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.4.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_file.unused_memory.q35: STARTED
 (4/4) Host_RHEL.m9.u4.ovmf.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.*.0.x86_64.io-github-autotest-qemu.sev_hotplug_mem.backend_file.unused_memory.q35: PASS (144.27 s)